### ISSUE TITLE
Arcana planets generating in vanilla stars (Arcana stars not generating)

### ISSUE THREAD

**Issue by NinCollin** [sleepySva/sva_arcana#2]
Perhaps this is simply an issue with my other mods, but I figured I would bring it to light anyway. I have Frackin Universe installed, and I decided I'd like to try out Arcana too. I do not have any other mods installed that effect planet/universe generation.

I'm using my current character and universe, and moving the starmap to ungenerated areas of the universe causes new Arcana planets to generate, but only in vanilla stars; the new Arcana stars do not seem to generate at all (only vanilla and FU stars.) 

I would assume this is not intentional. I can attach logs if necessary.

**Comment by sleepySva** [comment]
Hmm, I don't have any issues with planet / star generation on my end with FU installed, what mods are you using at the moment?

**Comment by NinCollin** [comment]
Here's the full list of mods I have (as well as a short description of changes for clarity)

- Boundless Space Stations (simply changes max size of player space stations)
- CuteBound (only changes graphics)
- Synthesizers (adds instruments to loot tables)
- xbawks (character extender)
- Skywind08's Anime Music Pack (just more songs)
- Frogg Villager Fix (changes Frogg villagers to be villager NPCs)
- CuteBound Arcana (only changes Nova Station venders)
- Frackin Universe
- Frackin Music
- Arcana

None of the other mods should be interfering with planet/star generation because they modify unrelated things. I'm going to try removing the other mods just in case as well as trying a fresh universe/character.

**Comment by NinCollin** [comment]
I removed everything except FU, Frackin Music, xbawks character extender, and Arcana, and I generated a fresh universe and character, but still had the issue.

However, as it turns out, updating FU (which was version 6.1.221, released Feb 15th of this year) allows the new Arcana stars to generate (with their respective planets.) However, Arcana planets still generating within vanilla stars as well. Is this intentional?
